apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -26,6 +27,34 @@ type ProductTutorialPageProps = {
   bodyConfig: ParseContentConfig;
 } & LayoutProps;
 
+export async function generateMetadata({
+  params,
+}: {
+  params: Params;
+}): Promise<Metadata> {
+  const productSlug = params?.productSlug;
+  const tutorialPath = params?.productTutorialPage?.join('/');
+  const tutorialProps = await getTutorial(productSlug, [tutorialPath]);
+  const { page } = tutorialProps;
+
+  const body = page.body;
+  const lines = body.split('\n').filter((line) => line !== '');
+  const title = lines[0].replace('# ', '');
+
+  return {
+    title,
+    openGraph: {
+      title,
+      url: page.path,
+    },
+    twitter: {
+      site: title,
+      card: 'summary',
+      creator: '@pagopa',
```

Comment:
What do you think to avoid duplication? I saw the `@pagopa` into the `twitter` block somewhere else